test/extended/machines/scale.go

```diff
@@ -26,7 +27,7 @@ const (
 	machineAPIGroup                = "machine.openshift.io"
 	machineSetOwningLabel          = "machine.openshift.io/cluster-api-machineset"
 	machineSetDeleteNodeAnnotaiton = "machine.openshift.io/cluster-api-delete-machine"
-	scalingTime                    = 12 * time.Minute
+	scalingTime                    = 15 * time.Minute
```

Comment:
As clearly seen you intentionally increased it. I'm asking just because of curiosity; what is the reason that?

Reply:
In my local test I started to have timeout failure, as it seems that the provisioning time somehow slightly increased, so I also retouched it to see it if it works in the CI